packages/scripts/src/application.ts

```diff
@@ -386,6 +458,8 @@ export class Application {
             publicConfigsRoute,
             overrideConfig,
             singleFeature,
+            createWebpackConfig: isExternal ? createWebpackConfigForExteranlFeature : createWebpackConfig,
```

Comment:
in order to be able to reuse `createWebpackConfigs` I needed to be able to provide the `createWebpackConfig` from outside